compose/ci.yaml

```diff
@@ -6,4 +6,6 @@ services:
     image: fuzzbench
     links:
       - queue-server
+    environment:
```

Comment:
I don't love that this file is called CI since it is really about end-to-end tests and we use CI for other things as well.

Reply:
Good point, let me rename it!